docs/how_to_add_a_publisher.md

```diff
@@ -333,14 +335,140 @@ def title(self) -> Optional[str]:
 
 ### Extracting Attributes with XPath and CSS-Select
 
-Sometimes you have to get information directly from the DOM of the HTML (in most cases the article text).
-To do so we suggest using [**lxml**](https://lxml.de/) and selectors like [`XPath`](https://lxml.de/xpathxslt.html) or [`CSSSelect`](https://lxml.de/cssselect.html).
-You can use both selectors on the `doc` attribute of `Precomputed`.
-If you use a selector in any of your attributes (which will be most likely the case when extracting the actual article text) you should precompute the selector as a class variable using the `XPath` respectively `CSSSelector` classes of lxml.
+When parsing the `ArticleBody`, or in certain scenarios, you need to directly obtain information from the [Document Object Model](https://en.wikipedia.org/wiki/Document_Object_Model) (DOM) of the HTML/XML.
```

Comment:
"Or in certain scenarios" should be left out or expanded upon.

Reply:
I agree, i will expand this.